src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -101,7 +101,37 @@ public void testVCFModeIsConsistentWithPastResults(final String inputFileName, f
             IntegrationTestSpec.assertEqualTextFiles(output, expected);
         }
     }
-    
+
+    /*
+     * Test that minimap2 data are supported and consistent with past results
+     */
+    public void testVCFModeMinimap2IsConsistentWithPastResults() throws Exception {
```

Comment:
this doesn't get run without adding the @test tag

Reply:
Fixed